CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v0.5.0 / 2016-11-01
+
+This release requires a storage wipe. It contains fundamental internal
+changes that came with implementing the high availability mode.
+
+* [FEATURE] Alertmanager clustering for high availability
+* [FEATURE] Garbage collection of old silences and notification logs
+* [CHANGE] New storage format
+* [CHANGE] Stricter silence semantics for consistent historical view
+
 ## v0.5.0-beta.0 / 2016-10-06
```

Comment:
Generally I would agree but both of these pre releases were out there for a month each. So I would want a more detailed changelog in this case as a user. But also wouldn't merging them.